lib/galaxy/model/unittest_utils/gxy_model_fixtures.py

```diff
@@ -895,6 +895,17 @@ def make_instance(*args, **kwds):
     return make_instance
 
 
```

Comment:
Both fixtures can be dropped (together with the mapping tests) - again, sorry about that!